### PR TITLE
ICP-11574 - query device for current status in response to WakeUpNotification

### PR DESCRIPTION
**Pull request by KKlimczukS**
ICP-11574 - query device for current status in response to WakeUpNotification

@greens @dkirker  Please, could You take a look at the code ?
@SmartThingsCommunity/srpol-pe-team

### CODE REVIEW

File: devicetypes/smartthings/zwave-smoke-alarm.src/zwave-smoke-alarm.groovy [hunk 1]
```diff
@@ -192,12 +192,18 @@ def zwaveEvent(physicalgraph.zwave.commands.wakeupv1.WakeUpNotification cmd, res
 	results << createEvent(descriptionText: "$device.displayName woke up", isStateChange: false)
 	if (!state.lastbatt || (now() - state.lastbatt) >= 56*60*60*1000) {
 		results << response([
+				zwave.notificationV3.notificationGet(notificationType: 0x01).format(),
+				"delay 2000",
```
Comment (dkirker) [inline devicetypes/smartthings/zwave-smoke-alarm.src/zwave-smoke-alarm.groovy]: I would use `delayBetween([], 2000)` here.

**Comment by greens** [comment]
I'm going to hold off on this until we've had someone else take a look at the zniffer logs.

**Comment by KKlimczukS** [comment]
> I'm going to hold off on this until we've had someone else take a look at the zniffer logs.

@greens 
I think that regardless of the result of looking at those logs we need that change in the code anyway, especially in case when:
Hub goes offline, meanwhile device reports smoke alarm, then hub goes online and after an hour device wakes up - and at this moment it has to be queried for its state.

**Comment by greens** [comment]
@KKlimczukS Alright. I see your point. Can you clean up the code in the way Donald described?

**Comment by KKlimczukS** [comment]
> @KKlimczukS Alright. I see your point. Can you clean up the code in the way Donald described?

@greens 
Sure, done.

**Review by greens — APPROVED** [review]

**Comment by greens** [comment]
Will let @dkirker take a look.

**Review by dkirker — APPROVED** [review]

**Pull request merged**